src/main/java/mate/academy/Main.java

```diff
@@ -2,15 +2,24 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.dao.impl.UserDaoImpl;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.User;
+import mate.academy.service.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.impl.AuthenticationServiceImpl;
+import mate.academy.service.impl.UserServiceImpl;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class Main {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws RegistrationException, AuthenticationException {
         MovieService movieService = null;
```

Comment:
The `movieService` is initialized as `null`. This will cause a `NullPointerException` when you try to call methods on it. Ensure that `movieService` is properly instantiated before use.